galleries/plot_types/basic/README.rst

```diff
@@ -1,6 +1,7 @@
 .. _basic_plots:
 
-Basic
------
+Pairwise data: ``(x, y)``
+-------------------------
 
-Basic plot types, usually y versus x.
+Plots of pairwise ``(x, y)``, functional ``f(x)=y``, and tabular
+``(var1, var2)`` data.
```

Comment:
What is the difference between pairwise and tabular?

I also have the feeling that "pairwise" in the title and in the explanation are used with slightly different semantics:
- Explanation: I assume that pairwise refers to the scatter plot, where you have data on two independent variables.
- Title: pairwise is more general in the sense that you have paired values but their relation can dependent or independent.